Automattic/WP-Job-Manager
# Introduce Save Draft button on job submission form

Fixes #505 

#### Changes proposed in this Pull Request:

* This adds a `Save Draft` button on the job submission form. It builds off of the functionality in #1647. 
* New filter: `submit_job_form_can_resume_later` (prefix matches other filters on submission form).
* Set button after Preview step so that visitors who pressed return/enter while in a text field would move through submission process.
* Added `secondary` class so that themes could make the button different/less prominent that main button.
* ~Note: HTML5 + server side validation must pass in order to save as a draft. We might be able to get around this for some fields, but we need it for fields related to registration. I think if we tackle this, it would be in a second phase.~ 4f7e426c0bd5d8ceefd863e70fc50c94b098e46c

To disable: `add_filter( 'submit_job_form_can_resume_later', '__return_false' );`

#### Testing instructions:

* Ensure jobs can be saved as drafts for logged in users or when user registration is enabled/forced. It shouldn't show up for guests or when users are unable to create accounts from the job submission page (job submission settings).
* Once saved, drafts should be accessible and resume-able from the `[job_dashboard]` page.


#### Proposed changelog entry for your changes:
* Feature: Allow job listings to be saved as drafts during the submission process.
